include/swift/Option/Options.td

```diff
@@ -1037,6 +1037,13 @@ def dwarf_version : Joined<["-"], "dwarf-version=">,
   Flags<[FrontendOption]>,
   HelpText<"DWARF debug info version to produce if requested">,
   MetaVarName<"<version>">;
+def dwarf_fission : Flag<["-"], "dwarf-fission">,
+  Flags<[FrontendOption]>,
+  HelpText<"Emit debug info as split-DWARF (Fission) in a separate .dwo file">;
+def emit_split_dwarf_path : Separate<["-"], "split-dwarf-output">,
```

Comment:
I think that `-dwarf-fission-path` would be a better option name. If you look at existing options, they match: `-emit-module -emit-module-path`.

Reply:
Done